tests/akstats/akstats_test.py

```diff
@@ -0,0 +1,94 @@
+import math
+
+from base_test import ArkoudaTest
+from context import arkouda as ak
+
+
+class StatsTest(ArkoudaTest):
+    def setUp(self):
+        ArkoudaTest.setUp(self)
+
+    def create_stat_test_pairs(self):
+        pairs = [
+            (
+                ak.array([10000000, 20000000, 30000000, 40000000, 50000000, 60000000, 70000000]),
+                ak.array([10000000, 20000000, 30000000, 40000001, 50000000, 60000000, 70000000]),
+            ),
+            (ak.array([10000000, 20000000, 30000000, 40000000, 50000000, 60000000, 70000000]), None),
+            (ak.array([44, 24, 29, 3]) / 100 * 189, ak.array([43, 52, 54, 40])),
+        ]
+        return pairs
+
+    def test_power_divergence(self):
+        from scipy.stats import power_divergence as scipy_power_divergence
+
+        from arkouda.akstats import power_divergence as ak_power_divergence
+
+        pairs = self.create_stat_test_pairs()
+
+        lambdas = [
+            "pearson",
+            "log-likelihood",
+            "freeman-tukey",
+            "mod-log-likelihood",
+            "neyman",
+            "cressie-read",
+        ]
+
+        ddofs = [0, 1, 2, 3, 4, 5]
+
+        for f_obs, f_exp in pairs:
+            for lambda0 in lambdas:
+                for ddof in ddofs:
+                    ak_power_div = ak_power_divergence(f_obs, f_exp, ddof=ddof, lambda_=lambda0)
+
+                    np_f_obs = f_obs.to_ndarray()
+                    np_f_exp = None
+                    if f_exp is not None:
+                        np_f_exp = f_exp.to_ndarray()
+
+                    scipy_power_div = scipy_power_divergence(
+                        np_f_obs, np_f_exp, ddof=ddof, axis=0, lambda_=lambda0
+                    )
+
+                    if math.isnan(ak_power_div.statistic):
+                        self.assertTrue(math.isnan(scipy_power_div.statistic))
+                    else:
+                        self.assertLess(
+                            abs(ak_power_div.statistic - scipy_power_div.statistic), 0.1 / 10**6
+                        )
+
+                    if math.isnan(ak_power_div.pvalue):
+                        self.assertTrue(math.isnan(scipy_power_div.pvalue))
+                    else:
+                        self.assertLess(abs(ak_power_div.pvalue - scipy_power_div.pvalue), 0.1 / 10**6)
```

Comment:
same comment about `np.allclose` here and in `test_chisquare`

Reply:
Will do.  Thanks!